doc/users/glossary.rst

```diff
@@ -0,0 +1,44 @@
+========
+Glossary
+========
+
+.. Note for glossary authors:
+   The glossary is primarily intended for Matplotlib's own concepts and
+   terminology, e.g. figure, artist, backend, etc. We don't want to list
+   general terms like "GUI framework", "event loop" or similar.
+   The glossary should contain a short definition of the term, aiming at
+   a high-level understanding. Use links to redirect to more comprehensive
+   explanations and API reference when possible.
+
+This glossary defines concepts and terminology specific to Matplotlib.
+
+.. glossary::
+
+    Figure
+        The outermost container for a Matplotlib graphic. Think of this as the
+        sheet of paper to draw on.
```

Comment:
```suggestion
        The outermost container for a Matplotlib graphic. Think of this as the
        canvas to draw on.
```
this is less a nitpick and more that the docs and several artists use the drawing area=canvas metaphor and we should keep that consistent